### ISSUE TITLE
Could you tell me how to solve the error: too few arguments?

### ISSUE THREAD

**Issue by deardream000** [yule-li/CosFace#12]
$ python test.py
usage: test.py [-h] [--network_type NETWORK_TYPE] [--fc_bn] [--prewhiten]
               [--save_model SAVE_MODEL] [--do_flip DO_FLIP]
               [--lfw_batch_size LFW_BATCH_SIZE]
               [--embedding_size EMBEDDING_SIZE] [--image_size IMAGE_SIZE]
               [--image_height IMAGE_HEIGHT] [--image_width IMAGE_WIDTH]
               [--lfw_pairs LFW_PAIRS] [--lfw_file_ext {jpg,png}]
               [--lfw_nrof_folds LFW_NROF_FOLDS] [--model_def MODEL_DEF]
               lfw_dir model
test.py: error: too few arguments

**Comment by deardream000** [comment]
@yule-li

**Comment by yule-li** [comment]
Please run ```test.sh``` to get test accuracy. If you want to run ```python test.py```, you can use the default arguments in ```test.sh```.

**Comment by deardream000** [comment]
Would you please tell me how to use the default arguments in test.sh? I can`t run test.sh.
test.sh: 20: test.sh: Syntax error: end of file unexpected (expecting "then")
Then what arguments do I need to change if I want to run python test.py?

**Comment by yule-li** [comment]
You can run command like this: ```CUDA_VISIBLE_DEVICES=1 python test/test.py dataset/lfw-112X96 models/model-20180309-083949.ckpt-60000 --lfw_file_ext jpg --network_type --embedding_size 512 --prewhiten --image_height 112 --image_width 96```, and modify ```dataset/lfw-112X96 models/model-20180309-083949.ckpt-60000``` to your saved model path if necessary.

**Comment by deardream000** [comment]
When I run test.sh
usage: test.py [-h] [--lfw_dir LFW_DIR] [--network_type NETWORK_TYPE]
               [--fc_bn] [--prewhiten] [--save_model SAVE_MODEL]
               [--do_flip DO_FLIP] [--lfw_batch_size LFW_BATCH_SIZE]
               [--embedding_size EMBEDDING_SIZE] [--model MODEL]
               [--image_size IMAGE_SIZE] [--image_height IMAGE_HEIGHT]
               [--image_width IMAGE_WIDTH] [--lfw_pairs LFW_PAIRS]
               [--lfw_file_ext {jpg,png}] [--lfw_nrof_folds LFW_NROF_FOLDS]
               [--model_def MODEL_DEF]
test.py: error: unrecognized arguments: dataset/lfw-112X96 models/model-20180309-083949.ckpt-60000

**Comment by deardream000** [comment]
Could you tell me your e-mail?

**Comment by wencoast** [comment]
@deardream000  @yule-li 
For using model-20180309-083949.ckpt-60000 pre-trained model, my setting as follows. It works for me. 
YOUR_DATA_DIR
YOUR_OWN_DIR/PretrainedModel/models/model-20180309-083949.ckpt-60000
--network_type
sphere_network
--prewhiten
--do_flip
True
--lfw_file_ext jpg
--embedding_size
512
--image_height 112
--image_width 96
